VoteCounter/src/VoteCount.cs

```diff
@@ -57,12 +63,40 @@ private void findVotes(Post post)
             {
                 if (isVoteValid(match.Groups[1].Value))
                 {
+                    
+                    string foundPlayer = new string('a',1);
```

Comment:
I had it blank before but I guess since string doesn't take 0 arguments and I wanted a new string I defaulted it there.